vcd/resource_vcd_vapp_vm_multinetwork_test.go

```diff
@@ -170,7 +170,7 @@ func TestAccVcdVAppVmMultiNIC(t *testing.T) {
 
 					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.name", "multinic-net"),
 					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.type", "org"),
-					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.is_primary", "true"),
+					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "network.1.is_primary", "false"),
```

Comment:
Looking at definition of `testAccCheckVcdVAppVmNetworkVmStep1` adaper #1 has `is_primary         = true`. Is this valid?

Reply:
I made update in HCL, 0 Net get false, 1 Net gets true.